graphql_compiler/tests/snapshot_tests/test_orientdb_match_query.py

```diff
@@ -13,8 +13,9 @@
 # pylint: disable=no-member
 
 
-def execute_graphql(schema, test_data, client):
+def execute_graphql(schema, test_data, client, sample_parameters=None):
```

Comment:
Let's not make this optional. Executed queries should explicitly say that they require no parameters. Making it optional is making it easy for people to forget to set this.